contents/handbook/growth/sales/how-we-work.md

```diff
@@ -6,34 +6,36 @@ showTitle: true
 
 > This page covers more of the operational detail of how our team generally works - for a broader overview of roles and responsibilities, visit the [overview page](/handbook/growth/sales/overview).
 
-## Main metrics for each role
+## Roles
 
-- Technical AE: new and expansion revenue in your book of business
-- RevOps: overall revenue from $20k+ segment
+We have three types of role:
+- Technical Account Executives - closing new business from [inbound](/handbook/growth/sales/inbound-sales) and [outbound](/handbook/growth/sales/outbound-sales) leads
+- Technical Account Managers - [expansion](/handbook/growth/sales/expansion-and-retention) from existing customers, closing new business from [product-led leads](/handbook/growth/sales/product-led-sales)
+- Technical Account Managers called Leon and Scott - [expansion](/handbook/growth/sales/expansion-and-retention) from a small set of very high potential existing customers
 
-Other metrics we generally care about:
-- Closed - won % from demo
-- Time to close
-- Lead volume in each segment
-- Weighted pipeline
+### Technical Account Executives
 
-## Book of business
+TAEs work with:
 
-### Account Executives
+- People email sales@ directly
+- People who book a demo via contact sales
+- Other triggers we see in product, supplemented by data from Clay
 
-Each AE is assigned up to 10 existing customer accounts to work with. Additionally, you will manage inbound leads as they are assigned to you in your territory. Overall, the hard cap on existing book + new leads is 15 accounts, so staying extremely focused is important. 
+When we start to generate cold outbound leads, these will be routed to TAEs to work with as well. Customers move off of a TAE to a TAM or CSM 3 months after closing on an annual contract - you have to ensure they are well set up, not just contract signed!
```

Comment:
I think 3 months is fine for now - next quarter I'd going to have a think about whether we can have a checklist of things (aka onboarding roadmap) that have to have been completed for an account to be handed over.  This encourages folks to actually move the needle with customers rather than just sit on a new customer for 3 months and do nothing.

Reply:
Yeah, how about we start with no cap for Q3 and see what happens? And over the next month or so, develop that onboarding checklist. 

I'd hope naturally anyway when @simfish85 and I review everyone's accounts each month we'd spot any massive outliers.